### DISCUSSION TITLE
Getting Error "Not authorized to send Apple events to System Events."

### DISCUSSION THREAD

**Discussion by jetblackrx89** [ruittenb/Spaceman#3] [Q&A] [answered]
I enabled Accessibility for Spaceman and relaunched it, but I am constantly getting the error "Not authorized to send Apple events to System Events." I think this is an Automation error and not Accessibility, but there's nothing in the Automation section of Privacy & Security that seems to have registered any previous activity from the app to permanently authorize it.

**Comment by chianyungcode** [comment]
![CleanShot 2024-12-28 at 23 07 00@2x](https://github.com/user-attachments/assets/702465f5-892d-43c3-988d-6e7eccac2596)
Same here

**Comment by ruittenb** [comment]
Hello @jetblackrx89 , I think you are right and Spaceman needs to have Automation permission too. This is what it looks like in my settings:

<img width="715" alt="Screenshot 2024-12-30 at 13 49 44" src="https://github.com/user-attachments/assets/9ab384d6-68a0-4e46-8adc-69547bd484fd" />

**Comment by ruittenb** [comment, marked as answer]
I made a fix in version 1.6.19.

**Reply by jetblackrx89** [reply]
Thanks, that did seem to fix it.
